x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ml/dataframe/analyses/ClassificationTests.java

```diff
@@ -299,16 +299,6 @@ public void testGetExplicitlyMappedFields() {
             equalTo(Collections.singletonMap("results.feature_importance", MapUtils.featureImportanceMapping())));
     }
 
-    public void testToXContent_GivenVersionBeforeRandomizeSeedWasIntroduced() throws IOException {
```

Comment:
As with outlier detection, the version here is in the config, not the current node version, so this should stay.